### PR TITLE
Announcement for the WildFly 23.0.2.Final Release

### PR DESCRIPTION
**Pull request by darranl**

### CODE REVIEW

File: _posts/2021-04-29-WildFly2302-Released.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-web-profile.adoc[Test Results - Web Profile - Java 11] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/347[Certification Request - Web Profile - Java 11]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform-jdk8.adoc[Test Results - Full Platform - Java 8] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/348[Certification Request - Full Platform - Java 8]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform.adoc[Test Results - Full Platform - Java 11] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/346[Certification Request - Full Platform - Java 11]
+
+We are hopeful this will mean the link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server can be declared compatible with Jakarta EE 9.1 at the time it is released for the web profile and full platform against both Java 8 and Java 11.
```
Comment (scottmarlow) [inline _posts/2021-04-29-WildFly2302-Released.adoc:29]: `web profile and full platform` should be `Web Profile and Full Platform`.

File: _posts/2021-04-29-WildFly2302-Released.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+== WildFly Preview and Jakarta EE 9
+
+The link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server still passes the Jakarta EE 9 for the TCKs link:https://github.com/wildfly/certifications/blob/EE9/WildFly_23.0.2.Final/jakarta-full-platform.adoc[Full Platform] and the link:https://github.com/wildfly/certifications/blob/EE9/WildFly_23.0.2.Final/jakarta-web-profile.adoc[Web Profile].  We have previously sent a link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/326[compatibility certification request for the Full Platform] for WildFly Preview 23.0.1.Final and a link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/315[compatibility certification request for the Web Profile] for WildFly Preview 23.0.0.Final.
+
+Once released Jakarta EE 9.1 will be the first Jakarta EE release that allows a server to certify using both Java 8 and Java 11 for both the web profile and the full platform.
```
Comment (bstansberry) [inline _posts/2021-04-29-WildFly2302-Released.adoc:20]: Once released,

File: _posts/2021-04-29-WildFly2302-Released.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform-jdk8.adoc[Test Results - Full Platform - Java 8] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/348[Certification Request - Full Platform - Java 8]
+ * link:https://github.com/wildfly/certifications/blob/EE9.1/WildFly_23.0.2.Final/jakarta-full-platform.adoc[Test Results - Full Platform - Java 11] / link:https://github.com/eclipse-ee4j/jakartaee-platform/issues/346[Certification Request - Full Platform - Java 11]
+
+We are hopeful this will mean the link:https://docs.wildfly.org/23/WildFly_and_WildFly_Preview.html[WildFly Preview variant] of the server can be declared compatible with Jakarta EE 9.1 at the time it is released for the web profile and full platform against both Java 8 and Java 11.
+
+This has been a busy couple of weeks working through the remaining issues to pass the candidate release of the Jakarta EE 9.1 TCK with particular thanks to Brian Stansberry, Scott Marlow, Richard Opalka, Jean-Francois Denise, Tomasz Adamski, Jim Ma, and Alessio Soldano for making it happen.
```
Comment (scottmarlow) [inline _posts/2021-04-29-WildFly2302-Released.adoc:31]: `Jakarta EE 9.1 TCK` should be `Jakarta EE Platform 9.1 TCKs`.

File: _posts/2021-04-29-WildFly2302-Released.adoc (hunk 1 above)
Comment (darranl) [reply _posts/2021-04-29-WildFly2302-Released.adoc]: Thanks, done

File: _posts/2021-04-29-WildFly2302-Released.adoc (hunk 2 above)
Comment (darranl) [reply _posts/2021-04-29-WildFly2302-Released.adoc]: Thanks, done

File: _posts/2021-04-29-WildFly2302-Released.adoc (hunk 3 above)
Comment (darranl) [reply _posts/2021-04-29-WildFly2302-Released.adoc]: Thanks, done

**Review by bstansberry — APPROVED** [review]

**Review by scottmarlow — APPROVED** [review]

**Pull request merged**